### PR TITLE
Fixed closed loop ramp rate

### LINKED ISSUE #50: Robot does wheelies
Robot is doing wheelies because it is accelerating too fast. To fix this, change the closed loop ramp rate.

### PR DESCRIPTION
**Pull request by LavaWaffle**
# **Purpose**: Prevent robot from doing wheelies 
## Files changed...
**Constants.java**: Changed closed loop ramp constant from 0.5d to 1.0d

### CODE REVIEW

File: src/main/java/frc/robot/Constants.java [hunk 1]
```diff
@@ -103,9 +103,8 @@
         public static final double STARTING_HEADING = Math.toRadians(0.0d);
         public static final int AUTO_LAUNCHER_RPM = 2160;
 
-        // Positions relative to location of driver station 👍 ➕ 🍶 🛰 
-        public static enum StartPositions 
-        {
+        // Positions relative to location of driver station 👍 ➕ 🍶 🛰 🤰 😘 😼 😹
```
Comment (bjmcternan) [inline src/main/java/frc/robot/Constants.java]: Do not put emojis in your code. These characters can be non-printable and cause horrid bugs.

File: src/main/java/frc/robot/Constants.java [hunk 2]
```diff
@@ -115,11 +114,12 @@
...
         }
     }
-    public static final class LauncherConstants{
+
+    public static final class LauncherConstants {
         public static final int LAUNCHER_CAN_ID = 6;
         public static final double TARGET_RPM_READY_THRESHOLD = 12.5;
         public static final int DEFAULT_TARGET_RPM = 2400;
-        public static final double CLOSED_LOOP_RAMPRATE = 0.5d;
+        public static final double CLOSED_LOOP_RAMPRATE = 1.0d;
```
Comment (bjmcternan) [inline src/main/java/frc/robot/Constants.java]: Why adjust the closed loop ramp for the launcher?

File: src/main/java/frc/robot/Constants.java [hunk 3]
```diff
@@ -60,11 +60,11 @@
         public static final double POSITION_KP = 0.016d;
         public static final double POSITION_KI = 0.0001d;
         public static final double POSITION_KD = 0.16d;
-        
-        public static final double CLOSED_LOOP_RAMP = 0.5;
+
+        public static final double CLOSED_LOOP_RAMP = 1.0d;
```
Comment (bjmcternan) [inline src/main/java/frc/robot/Constants.java:64]: Does this actually work? We're using percent drive not motion magic

**Review by bjmcternan — CHANGES_REQUESTED** [review]
100% do not put emojis in your code. It can cause insanely difficult to fix bugs if any of them are non-printable by the IDE.

Also, was this tested? We're using percent drive not motion magic. I don't think we should change the closed loop ramp of the launcher

File: src/main/java/frc/robot/Constants.java (hunk 3 above)
Comment (LavaWaffle) [reply src/main/java/frc/robot/Constants.java:64]: It may work, we weren't able to test because mech was using robot

Comment (bjmcternan) [reply src/main/java/frc/robot/Constants.java:64]: Oh. This is not ready for PR then. Let's decline this for now until we test it

**Comment by bjmcternan** [comment]
Still need to test.

**Pull request closed without merging**